src/errormessages.jl

```diff
@@ -0,0 +1,44 @@
+
+"""
+This is borrowed from [DataFrames.jl](). 
+Return between 0 and 2 names from `colnames` closest to `name`.
+`colnames` : some iterable collection of symbols
+"""
+function fuzzymatch(colnames, name::Symbol)
+        ucname = uppercase(string(name))
+        dist = [(levenshtein(uppercase(string(x)), ucname), x) for x in colnames]
+        sort!(dist)
+        c = [count(x -> x[1] <= i, dist) for i in 0:2]
+        maxd = max(0, searchsortedlast(c, 8) - 1)
+        return [s for (d, s) in dist if d <= maxd]
+end
+
+"""
+Return a nice-ish error message if the Symbol `name` isn't a column name in `table`, otherwise a zero-length string.
+"""
+function checkcol(table, name :: Symbol)::String
```

Comment:
Same as below...I don't think the type assert is really necessary here since it's sufficiently clear from the (short) function body that a string will be returned
```suggestion
function checkcol(table, name::Symbol)
```